api/hrlh.rb

```diff
@@ -1,5 +1,6 @@
 require 'grape'
 require 'orchestrate'
+require 'time'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.